backend/climateconnect_api/utility/email_setup.py

```diff
@@ -214,7 +273,27 @@ def send_password_link(user, password_reset_key, hub_url=None):
     )
 
 
-def send_feedback_email(email, message, send_response):
+def send_feedback_email(email, message, send_response, user_agent=None, path=None):
+    # Parse user agent to get readable browser and OS info
+    user_agent_info = get_formatted_user_agent_info(user_agent) if user_agent else None
+
+    # Build template variables with parsed info
+    variables = {
+        "text": str(message),
+        "sendReply": str(send_response),
+        "email": str(email if email else ""),
+        "path": str(path if path else ""),
+    }
+
+    # Add parsed user agent info if available
+    if user_agent_info:
+        variables["browser"] = user_agent_info["browser"]
+        variables["os"] = user_agent_info["os"]
+        variables["device"] = user_agent_info["device"] or ""
+        variables["userAgent"] = user_agent_info["raw"]  # Keep raw for debugging
+    else:
+        variables["userAgent"] = str(user_agent if user_agent else "")
```

Comment:
The Mailjet template is fault-tolerant, so we don't have to provide empty values if we don't have it.